README.md

```diff
@@ -31,22 +31,35 @@ Or install it yourself as:
 ```ruby
 require 'plotlyrb'
 
-data = {
-  :x => ['2013-10-04 22:23:00', '2013-11-04 22:23:00', '2013-12-04 22:23:00'],
-  :y => [1, 3, 6],
-  :type => 'scatter',
-  :mode => 'markers',
-}
+# NOTE: data argument must be an array of traces
+data = [
+  {
+    :x => ['2013-10-04 22:23:00', '2013-11-04 22:23:00', '2013-12-04 22:23:00'],
+    :y => [1, 3, 6],
+    :type => 'scatter',
+    :mode => 'markers',
+  },
+  {
+    :pokemon => ['pikachu', 'squirtle', 'charmander'],
+    :number_caught => [1, 3, 9],
+    :type => 'scatter'
+  }
+]
 
 layout = {
   :xaxis => { :title => 'times' },
   :yaxis => { :title => 'bigfoot sightings', :range => [0, 7] },
 }
 
+plot_spec = {
+  :figure => {:data => data},
+  :layout => layout,
+  :format => :svg
+}
```

Comment:
@ajmccluskey I think this needs to be
```
plot_spec = {
   :figure => {:data => data, :layout => layout},
   :format => :svg
 }
```
At least, that seems to work for my Dta gem and some other code I'm working on at the moment.